gofedlib/distribution/data/ip2package_mapping.json

```diff
@@ -1,5 +1,4 @@
 [{"ipprefix": "github.com/GoogleCloudPlatform/kubernetes","package":"kubernetes"},
-{"ipprefix": "k8s.io/kubernetes","package":"kubernetes"},
```

Comment:
this is needed, otherwise there is no way to translate kubernetes prefix into kubernetes package